resources/shared/kubernetes/ingress-controller.ts

```diff
@@ -25,6 +25,7 @@ export const ingress = new k8s.helm.v3.Chart(
       ingressController: {
         config: {
           email: systemEmail,
+          onDemandTLS: true,
```

Comment:
This makes so that SSL certificates are generated when the first request is done to a new URL (related to wildcard setup).